### DISCUSSION TITLE
Hundreds of volumes without assigned collection

### DISCUSSION THREAD

**Discussion by divanikus** [seaweedfs/seaweedfs#2165] [General]
**Describe the bug**
I see hundreds of volumes without any assigned bucket or collection (see screenshot). Most of them have 0 or a couple of files in them. What are they? How can I get rid of them, because they occupy a lot of volume count limit?
![image](https://user-images.githubusercontent.com/3438036/123785932-c50ae980-d8e1-11eb-8b3f-02d7c43d05f4.png)

**System Setup**
- OS version: Debian 10
- output of `weed version`: version 30GB 2.55 05af54a linux amd64

**Comment by chrislusf** [comment]
Maybe you pre-allocated the volumes?
You can delete them by removing the `<volumeId>.idx` and `<volumeId>.dat` files. And restart.

**Reply by divanikus** [reply]
The problem is, I didn't pre-allocated them by myself. They appeared by the normal operation of seaweedfs. We only use it through S3 gateway.
